### DISCUSSION TITLE
why is just copy possible Text Selection Toolbar

### DISCUSSION THREAD

**Discussion by shahram7** [ReadYouApp/ReadYou#729] [Q&A]
Hi why it's just copy possible in Text Selection Toolbar or the Contextual Action Bar? I would like to be able to use my dictionaries when I need. 
and It's a little bit annoying to copy the word and switch to the dictionary app each time! 
![Screenshot_20240603_235101_Read You](https://github.com/Ashinch/ReadYou/assets/34509793/b9568aa3-c85d-437e-a01e-e84ef6f4a895)

![Screenshot_20240603_234428_Brave](https://github.com/Ashinch/ReadYou/assets/34509793/7e67751c-1e0d-4c7f-87c3-e618fed09e42)
![Screenshot_20240603_234439_Brave](https://github.com/Ashinch/ReadYou/assets/34509793/7b7349d8-3d17-46a9-a38b-2ea8a76dd7ba)

**Comment by shuvashish76** [comment]
Duplicate of #117
